app/build.gradle

```diff
@@ -244,7 +244,9 @@ dependencies {
     implementation 'com.github.wooplr:Spotlight:1.3'
     implementation 'com.google.code.findbugs:jsr305:3.0.2'
     implementation 'com.github.nextcloud-deps:ChatKit:0.4.2'
-
+    testImplementation 'junit:junit:4.13.2'
```

Comment:
`junit` and `mockito` are duplicated, see https://github.com/nextcloud/talk-android/pull/3280/files#diff-51a0b488f963eb0be6c6599bf5df497313877cf5bdff3950807373912ac1cdc9R290 for the original entries and please then move the `mockito-android` dependency down there, so the test dependencies are grouped together.